src/test/java/capstone/mfslbackend/service/TeamServiceTest.java

```diff
@@ -86,7 +90,19 @@ public void testCreateTeamsInLeague_SuccessExists() throws IOException {
         TeamsContainer teamsContainer = new TeamsContainer();
         List<TeamsResponse> teamsResponses = new ArrayList<>();
 
-        TeamResponse teamResponse1 = new TeamResponse();
+
+//        this.id = id;
+//        this.name = name;
+//        this.logo = logo;
+//        this.update = update;
+//        this.winner = winner;
+//        this.code = code;
+//        this.country = country;
+//        this.founded = founded;
+//        this.national = national;
+
+
+        TeamResponse teamResponse1 = new TeamResponse(teamId, teamName, teamName, teamName, true, teamName, teamName, 1, true);
         teamResponse1.setId(teamId);
```

Comment:
you should remove these because you're just duplicating code, you already set them by adding them to the constructor